README.md

```diff
@@ -497,7 +497,7 @@ public abstract class HockeyPlayer implements HockeyPlayerModel {
   public static final RowMapper<NamesForNumber> SELECT_NAMES_MAPPER =
       FACTORY.select_namesMapper(AutoValue_HockeyPlayer_Names::new);
 
-  public List<Names> names(SQLiteDatabase) {
+  public List<Names> names(SupportSQLiteDatabase) {
```

Comment:
Weird that there's no parameter here..

Reply:
I'm going to revert all the README changes. It'll only confuse users.